extensions/notebook/src/jupyter/jupyterServerManager.ts

```diff
@@ -54,6 +54,10 @@ export class LocalJupyterServerManager implements nb.ServerManager, vscode.Dispo
 		return this._onServerStarted.event;
 	}
 
+	public get jupyterInstallation(): JupyterServerInstallation {
```

Comment:
>JupyterServerInstallation [](start = 35, length = 25)

This should be returning `JupyterServerInstallation | undefined` since those are the possible return values